vector/src/main/java/im/vector/app/features/home/room/detail/composer/RichTextComposerLayout.kt

```diff
@@ -188,6 +190,16 @@ internal class RichTextComposerLayout @JvmOverloads constructor(
         views.plainTextComposerEditText.addTextChangedListener(
                 TextChangeListener({ callback?.onTextChanged(it) }, { updateTextFieldBorder(isFullScreen) })
         )
+        ViewCompat.setOnReceiveContentListener(
```

Comment:
I was wondering if we should unregister all of these listeners but I suppose they will be destroyed with the view as they're owned by the view?